package.json

```diff
@@ -1,14 +1,41 @@
 {
-	"name": "new-dev-setup",
-	"version": "1.0.0",
-	"main": "index.js",
-	"repository": "https://github.com/gsandf/new-dev-setup.git",
-	"author": "Randy Uebel <ruebel@gsandf.com> (http://www.gsandf.com)",
-	"license": "MIT",
-	"scripts": {
-		"start": "autoreload-server -w \"**/**.{html,css,js}\" ./docs 8008"
-	},
-	"dependencies": {
-		"simple-autoreload-server": "^0.2.7"
-	}
+  "name": "new-dev-setup",
+  "version": "2.0.0",
+  "main": "dist/setup.js",
+  "module": "src/index.js",
+  "repository": "https://github.com/gsandf/new-dev-setup.git",
+  "author": "GS&F Devs <mindreactor@gsandf.com> (https://gsandf.com/)",
+  "license": "MIT",
+  "files": [
+    "src/**",
+    "dist/**"
+  ],
+  "eslintConfig": {
+    "extends": "gsandf",
+    "parser": "babel-eslint"
+  },
```

Comment:
Out of curiosity, why did you opt for setting this in the `package.json` instead of it's own file?

Reply:
No reason